tests/code/test_addRemoveInterest.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from app.logic.addRemoveInterest import addRemoveInterest
+from peewee import IntegrityError, DoesNotExist
+
+@pytest.mark.integration
+def test_addRemoveInterest():
+    user = "ramsayb2"
+    program_id = 1
+

```

Comment:
you are using a variable named `addInterest` for all of these, which is confusing. I would change it to `result` so it matches how it is used